### PR TITLE
[fix](cloud) port CloudLoadManager#removeCopyJobs() to master

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes

Remove completed CopyJobs when copy job count per table exceeding `cloud_max_copy_job_per_table`.
To prevent memory leak.

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java [hunk 1]
```diff
@@ -484,6 +485,7 @@ private void removeLoadJobIf(Predicate<LoadJob> pred) {
                     removeJobNum++;
                 }
             }
+            removeCopyJobs();
```
Comment (HHoflittlefish777) [inline fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java:488]: the logic should in cloud load manager

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Pull request merged**